src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -2228,7 +2259,36 @@ async def CommissionOnNetwork(self, nodeId: int, setupPinCode: int,
                     self.devCtrl, self.pairingDelegate, nodeId, setupPinCode, int(filterType), str(filter).encode("utf-8") if filter is not None else None, discoveryTimeoutMsec)
             )
 
-            return await asyncio.futures.wrap_future(ctx.future)
+            res = await asyncio.futures.wrap_future(ctx.future)
+
+            # If RCAC data is wanted, attempt to pull the result
+            if get_rcac:
```

Comment:
can you get rid of this and just have the caller call get_rcac directly? I'd rather keep the API for commissioning the same across all the network transports as much as possible.

Reply:
I have moved the get_rcac() to a standalone function that can now be called from test modules, such as self.th2.get_rcac() once commissioning has been completed.